documentation/aws-account-and-identity-architecture.md

```diff
@@ -12,6 +12,18 @@ The applications and project teams develop and operate independently, have limit
 implementation technologies. Helpful Engineering has neither the time nor the staff to support high touch, high coordination
 solutions.
 
+
+**Global vs Hierarchical Identity Namespace**
+
+AWS' Identity and Access Management service provides the primitives to implement fine-grained access control to 
++3,500 AWS service APIs and resources. However, IAM provides no real opinion on how to model and implement complex
+access control structures required by large organizations. The problem of ensuring Team A only has access to Team A's 
+resources within an AWS account is left to the AWS user. IAM provides a flat, global namespace and definition structure
+within an Account for identities and policies. By contrast, GCP and Azure provide hierarchical modeling mechanisms and
+inheritance that simplifies implementing more complex relationships.
+
+We will describe our own solution next.
```

Comment:
```suggestion
```

Maybe this transition could be removed without negatively impacting the document structure.